snippets/csharp/VS_Snippets_CLR/CryptoWalkThru/cs/Form1.cs

```diff
@@ -78,8 +78,10 @@ private void buttonCreateAsmKeys_Click(object sender, System.EventArgs e)
 private void buttonEncryptFile_Click(object sender, System.EventArgs e)
 {
     if (rsa == null)
-        MessageBox.Show("Key not set.");
-    else
+            {
+                MessageBox.Show("Key not set.");
+            }
+            else
     {
```

Comment:
and here. It seems that there will be more occurrences of the indentation issue. I'm not going to point more of them.